org.opent2t.sample.hub.superpopular/com.wink.hub/js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "opent2t-wink-hub",
+  "version": "1.0.0",
+  "description": "Hub schema and helper modules for Open Translators to Things",
```

Comment:
For consistency with other translators, I suggest this should just be "Wink Hub". We are using this field as the "friendly human readable name of this thing".